distributed/deploy/tests/test_spec_cluster.py

```diff
@@ -416,6 +415,9 @@ def __str__(self):
 
     __repr__ = __str__
 
+    def __await__(self):
+        return self.start().__await__()
+
```

Comment:
This `MultiWorker` is a bit strange since it isn't a Server although `Worker`s are always `Server`s